plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -341,9 +341,9 @@ def move_to_target_dir(download_path) -> str:
                     error_msg = f"Expected a single directory, found: {paths}"
                     raise DownloadError(error_msg)
                 package_path = paths.pop()
-
-            package_path.rename(target_dir / package_path.name)
-            return str(package_path)
+            # move not rename cause can be on different filesystems
+            shutil.move(package_path, target_dir / package_path.name)
+            return str(target_dir / package_path.name)
```

Comment:
@solarw can you explain the issue with [Path.rename](https://docs.python.org/3/library/pathlib.html#pathlib.Path.rename)? An example would be nice.

Reply:
@Karrenbelt 
rename works only inside the same filesystem
https://man7.org/linux/man-pages/man2/rename.2.html 
```
https://man7.org/linux/man-pages/man2/rename.2.html
```


in my particular case `/tmp` is a tmpfs in memory. so rename from `/tmp` to `/home/...` does not work
data has to be copied from one filesystem to another